plan9/buildinfo

```diff
@@ -1 +1 @@
-p9pvers = 5.008
```

Comment:
We're at 5.32.0 now ;-)

Reply:
Yep, it's been bitrotten for at least that long!

Reply:
Ah, I see, I inserted 5.30.2 when it should have been 5.32.0. Fixed.